From: Simon McVittie <smcv@debian.org>
Date: Thu, 26 Jul 2018 09:55:34 +0100
Subject: Remove test for GNOME#753605

It relies on a non-free JPEG that happens to exhibit the bug.
---
 tests/meson.build   |  1 -
 tests/pixbuf-jpeg.c | 23 -----------------------
 2 files changed, 24 deletions(-)

diff --git a/tests/meson.build b/tests/meson.build
index aaa1817..21f0751 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -86,7 +86,6 @@ test_data = [
   'bug143608-comment.jpg',
   'bug725582-testrotate.jpg',
   'bug725582-testrotate.png',
-  'bug753605-atsize.jpg',
   'cve-2015-4491.bmp',
   'large.png',
   'large.jpg',
diff --git a/tests/pixbuf-jpeg.c b/tests/pixbuf-jpeg.c
index d03384e..ea1d62e 100644
--- a/tests/pixbuf-jpeg.c
+++ b/tests/pixbuf-jpeg.c
@@ -115,28 +115,6 @@ test_comment(void)
   g_object_unref (ref);
 }
 
-static void
-test_at_size (void)
-{
-  GError *error = NULL;
-  GdkPixbuf *ref;
-
-  if (!format_supported ("jpeg") || !format_supported ("png"))
-    {
-      g_test_skip ("format not supported");
-      return;
-    }
-
-  ref = gdk_pixbuf_new_from_file (g_test_get_filename (G_TEST_DIST, "bug753605-atsize.jpg", NULL), &error);
-  g_assert_no_error (error);
-  g_object_unref (ref);
-
-  ref = gdk_pixbuf_new_from_file_at_size (g_test_get_filename (G_TEST_DIST, "bug753605-atsize.jpg", NULL),
-					  50, 50, &error);
-  g_assert_no_error (error);
-  g_object_unref (ref);
-}
-
 int
 main (int argc, char **argv)
 {
@@ -146,7 +124,6 @@ main (int argc, char **argv)
   g_test_add_func ("/pixbuf/jpeg/type9_rotation_exif_tag", test_type9_rotation_exif_tag);
   g_test_add_func ("/pixbuf/jpeg/bug775218", test_bug_775218);
   g_test_add_func ("/pixbuf/jpeg/comment", test_comment);
-  g_test_add_func ("/pixbuf/jpeg/at_size", test_at_size);
 
   return g_test_run ();
 }
